drivers/crypto/crypto_mtls_shim.c

```diff
@@ -103,36 +104,55 @@ int mtls_ecb_decrypt(struct cipher_ctx *ctx, struct cipher_pkt *pkt)
 
 int mtls_cbc_encrypt(struct cipher_ctx *ctx, struct cipher_pkt *pkt, u8_t *iv)
 {
-	int ret;
+	int ret, iv_bytes;
+	u8_t *p_iv, iv_loc[16];
```

Comment:
I cannot remember why iv is of 16bytes fixed length here, is it because of cbc or because of mbedtls? 

Could you detail? (note: I know next to nothing about crypto)

Reply:
In AES-CBC, IV length is always 1 block size (128 bits) regardless of key lengths (AES-128, AES-256 etc.,) .

Reply:
ok